resources/css/_variables.css

```
@@ -24,6 +24,19 @@
     --theme-color-secondary-800: 73 76 94; /* #494c5e */
     --theme-color-secondary-900: 49 51 63; /* #31333f */
 
+    /* Dark Mode Gray */
+    --theme-color-dark-secondary-50: 227 232 234; /* #E3E8EA */
```

```suggestion
    --theme-color-dark-50: 227 232 234; /* #E3E8EA */
```

personally I'd call it just `dark` as it's not a replacement secondary color; it's used in addition to the regular palette